vendor/github.com/hashicorp/nomad/api/event_sink.go

```diff
@@ -52,6 +52,6 @@ func (e *EventSinks) Register(eventSink *EventSink, w *WriteOptions) (*WriteMeta
 	return wm, nil
 }
 
-func (e *EventSinks) Deregister(name string, w *WriteOptions) (*WriteMeta, error) {
-	return e.client.delete("/v1/event/sink/"+name, nil, w)
+func (e *EventSinks) Deregister(id string, w *WriteOptions) (*WriteMeta, error) {
```

Comment:
We shouldn't have this floating around after a `make sync` at this point and it looks like there's a merge conflict... can you rebase on `master`?